apps/minaintyg/src/pages/Certificates/components/CertificateListOrder.tsx

```diff
@@ -0,0 +1,33 @@
+import { classNames } from '@frontend/components'
+import { ReactEventHandler, ReactNode } from 'react'
+
+export type SortingOrder = 'ascending' | 'descending'
+
+export function SortingButton({ active, onClick, children }: { active: boolean; onClick: ReactEventHandler; children: ReactNode }) {
```

Comment:
This can be extracted into its own component

Reply:
There is a linting rule that could be placed to enforce this pattern https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-multi-comp.md. perhaps we should activate it, what do you think @ChrilleSigerhed ?

Reply:
I think that sounds like a good idea, I agree with the statement

```Declaring only one component per file improves readability and reusability of components.```